specs/swagger.yml

```diff
@@ -1689,6 +1732,69 @@ paths:
         '401':
           $ref: '#/responses/unauthorizedResponse'
 
+  '/identities/{id}/edge-routers':
+    parameters:
+      - $ref: '#/parameters/id'
+    get:
+      summary: List accessible edge-routers
+      description: |
+        Retrieves a list of edge-routers that the given identity may use to access services. Supports filtering, sorting, and pagination. Requires admin access.
+      security:
+        - ztSession: []
+      tags:
+        - Identity
+      operationId: listIdentityEdgeRouters
+      responses:
+        '200':
+          $ref: '#/responses/listEdgeRouters'
+        '404':
+          $ref: '#/responses/notFoundResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+
+  '/identities/{id}/services':
+    parameters:
+      - $ref: '#/parameters/id'
+    get:
+      summary: List accessible services
+      description: |
+        Retrieves a list of services that the given identity has access to. Supports filtering, sorting, and pagination. Requires admin access.
+      security:
+        - ztSession: []
+      tags:
+        - Identity
+      operationId: listIdentityServices
+      responses:
+        '200':
+          $ref: '#/responses/listEdgeRouters'
+        '404':
+          $ref: '#/responses/notFoundResponse'
+        '401':
+          $ref: '#/responses/unauthorizedResponse'
+
+  '/identities/{id}/policy-advice/{serviceId}':
+    parameters:
+      - $ref: '#/parameters/id'
+      - $ref: '#/parameters/serviceId'
+    get:
+      summary: Analyze policies relating the given identity and service
+      description: |
```

Comment:
I find this site helpful when dealing with long strings in YAML. YAML is much better at them than JSON.

https://yaml-multiline.info/

Lots of little knobs to turn.